src/MyClassSeventh1.java

```diff
@@ -0,0 +1,28 @@
+import java.util.Arrays;
+
+public class MyClassSeventh1 {
+    public static void divideArray(int[] array, int divisor) {
+        if (divisor == 0) {
```

Comment:
You catch exception via if statement. It's ok
But it was expected that you would do it using try/catch and it can throw ArithmeticException. You've implemented it but ArithmeticException has never been thrown because you've already caught it via if